packages/ui/tsup.config.ts

```diff
@@ -40,6 +40,7 @@ export default defineConfig(overrideOptions => {
       PACKAGE_NAME: `"${name}"`,
       PACKAGE_VERSION: `"${version}"`,
       __DEV__: `${!isProd}`,
+      'import.meta.vitest': 'undefined',
```

Comment:
Defining this to `undefined` during build allows rollup to correctly tree shake the test code out of the source files.